src/gt4py/next/program_processors/runners/dace_fieldview/gtir_to_sdfg.py

```diff
@@ -405,11 +407,7 @@ def visit_Lambda(
         # that is only the symbols that are used in the context of the lambda node
```

Comment:
```suggestion
        # that are only the symbols that are used in the context of the lambda node
```
I am not that good at grammar but I think it should be plural here.